src/code/padsetup.c

```diff
@@ -0,0 +1,30 @@
+#include <global.h>
+
+s32 PadSetup_Init(OSMesgQueue* mq, u8* outMask, OSContStatus* status) {
+    s32 ret;
+    s32 i;
+
+    *outMask = 0xFF;
+    ret = osContInit(mq, outMask, status);
+    if (ret != 0) {
+        return ret;
+    }
+    if (*outMask == 0xFF) {
+        if (osContStartQuery(mq) != 0) {
+            return 1;
+        }
+        osRecvMesg(mq, NULL, OS_MESG_BLOCK);
+        osContGetQuery(status);
+
+        *outMask = 0;
+        for (i = 0; i < 4; i++) {
+            if (status[i].errno == 0) {
+                if (status[i].type == CONT_TYPE_NORMAL) {
+                    *outMask |= 1 << i;
+                }
+            } else {
+            }
```

Comment:
```suggestion
            }
```

Reply:
The empty else is required to match.

Reply:
Required how? What does it change in codegen?

Reply:
The empty `else` adds a branch instruction.

Reply:
It's most likely a fake match but I guess there is no way to know for sure. Might be worth trying a switch on errno or something slightly different tho, like an early continue.

Reply:
Using a switch worked.